NEWS.md

```diff
@@ -1,4 +1,10 @@
 ## formatters 0.4.1.9001
+ * font_size is now 12 across the board, whereas before it was inconsistent between 8 and 12
+ * margins are now correctly interpreted as inches when specified (including the default)
+ * new `export_as_txt` function which can operate on any object with an applicable `matrix_form` method (including pagination)
+ * new `paginate_indices` and `paginate_to_mpfs` functions
```

Comment:
```suggestion
 * New `export_as_txt` function which can operate on any object with an applicable `matrix_form` method (including pagination).
 * new `paginate_indices` and `paginate_to_mpfs` functions.
```

Reply:
It doesn't make sense to me to change one line starting with New but not change the very next entry which starts the same way. I don't really care which way we do it, but if we're going to care about this kind of thing, we should be consistent.